components/TopBarProfileMenu.js

```diff
@@ -228,25 +232,32 @@ class TopBarProfileMenu extends React.Component {
     }
 
     return (
-      <div className="LoginTopBarProfileButton">
-        {status === 'loading' && (
-          <P color="#D5DAE0" fontSize="1.4rem" px={3} py={2} display="inline-block">
-            <FormattedMessage id="loading" defaultMessage="loading" />
-            &hellip;
-          </P>
-        )}
+      <React.Fragment>
+        <div className="LoginTopBarProfileButton">
+          {status === 'loading' && (
+            <P color="#D5DAE0" fontSize="1.4rem" px={3} py={2} display="inline-block">
+              <FormattedMessage id="loading" defaultMessage="loading" />
+              &hellip;
+            </P>
+          )}
 
-        {status === 'loggingout' && (
-          <P color="#D5DAE0" fontSize="1.4rem" px={3} py={2} display="inline-block">
-            <FormattedMessage id="loggingout" defaultMessage="logging out" />
-            &hellip;
-          </P>
-        )}
+          {status === 'loggingout' && (
+            <P color="#D5DAE0" fontSize="1.4rem" px={3} py={2} display="inline-block">
+              <FormattedMessage id="loggingout" defaultMessage="logging out" />
+              &hellip;
+            </P>
+          )}
 
-        {status === 'loggedout' && <LoginBtn />}
+          {status === 'loggedout' && <LoginBtn />}
 
-        {status === 'loggedin' && this.renderLoggedInUser()}
-      </div>
+          {status === 'loggedin' && this.renderLoggedInUser()}
+        </div>
+        <NewsAndUpdatesModal
```

Comment:
That can work for the MVP, but it does not make much sense to display `NewsAndUpdatesModal` in `TopBarProfileMenu`. What will happen when we'll want to add another trigger to display the modal in another component?

One common approach to this (when there's no global state provider like redux) is to have the component up in the tree (somewhere like `pages/_app.js`) and to provide a global hook (eg. `useNewsAndUpdateModal`) with a helper (eg. `showNewsAndUpdateModal`) to display it. Not very different from what we do with `GlobalToasts`.

Reply:
@Betree : Thanks, yes you are right; didn't think about that somehow. I've done this as well at, 2ddf3bb. Let me know if you see any further issues. 😄